lib/raven/interfaces/exception.rb

```diff
@@ -11,6 +11,7 @@ def self.sentry_alias
     def to_hash(*args)
       data = super(*args)
       data[:values] = data[:values].map(&:to_hash) if data[:values]
+      data[:values].freeze
```

Comment:
Removes exceptions from sanitizer path. RemoveStacktraceProcessor still works, of course.